docs/website/docs/plus/getting-started/advanced_tutorial.md

```diff
@@ -0,0 +1,305 @@
+---
+title: Advanced Project tutorial
+description: Using the dlt+ cli commands to create and manage dlt+ Project
+keywords: [command line interface, cli, dlt init, dlt+, project]
+---
+
+import Link from '../../_plus_admonition.md';
+
+<Link/>
+
+This tutorial will introduce you to advanced dlt project features. You will learn how to:
+
+* Understand how dlt+ creates implicit dlt entites if they are not defined and how to prevent this
+* Learn how to use dlt+ datasets to restrict access to certain datasets and destinations
+* Learn how to interact with the dlt+ project context and entities from code
+* Create a packaged dlt+ project which you can distribute to other stakeholders in your company
+
+## Prerequisites
+
+To follow this tutorial, make sure:
+
+- dlt+ is set up according to the [installation guide](./installation.md)
+- you're familiar with the [core concepts of dlt](../../reference/explainers/how-dlt-works.md)
+- you have completed the [basic project tutorial](./tutorial.md)
+
+## Implicit entities
+
+By default, dlt+ will create implicit entities when they are requested by the user or the executed code. Let's create an example project again with `dlt project init arrow duckdb --package my_dlt_project`.
```

Comment:
It'd be easier to copy the command for the reader if you use code block for `dlt project init arrow duckdb --package my_dlt_project`